ehitchhiking-ui/src/app/shared/components/trip/trip.component.ts

```diff
@@ -16,4 +19,15 @@ export class TripComponent implements OnInit {
 	makeFavorite() {
 		this.trip.isFavorite = !this.trip.isFavorite;
 	}
+
+	rating(){
```

Comment:
bad naming: rating. Pls rename to toggleRating

Reply:
fixed